ControlRoomApplication/ControlRoomApplication/Controllers/RadioTelescopeControllers/RadioTelescopeController.cs

```diff
@@ -122,6 +134,10 @@ public bool CancelCurrentMoveCommand()
         /// </summary>
         public bool ShutdownRadioTelescope()
         {
+            RadioTelescope.PLCDriver.Bring_down();
+            snowDumpTimer.Stop();
+            snowDumpTimer.Dispose();
+
             return MoveRadioTelescopeToOrientation(MiscellaneousConstants.Stow);
```

Comment:
Additionally, it would be good to comment out the movement to stow position after moving it to the top, because we don't want to do this for our hardware testing.

A GitHub issue and additional category for "Final Production Changes" should be made to uncomment this out, just so it is tracked in the issue tracker and we don't forget to do it.